lib/index.js

```diff
@@ -372,6 +394,28 @@ function opt (option) {
   }
 };
 
+/* chan and rmchan sockopt methods. only relevant for subscription sockets */
+Socket.prototype.chan       = function(){
+  var args = arguments;
+  var channels = args.length;
+  while(channels--) {
```

Comment:
I personally would understand the code, and it would look cleaner to me with less of it, but what about someone new to the project? 

Is short `var` more legible than a descriptive `var`?

As I recall ZeroMQ is somewhat opinionated about that, of course that's not necessarily the style of libnanomsg, whose naming conventions are more aligned with POSIX standards than probably anything else.